Core_Framework/NUnitTestSetup/NUnit_Test_Setup.cs

```diff
@@ -0,0 +1,66 @@
+﻿using AventStack.ExtentReports;
+using Core_Framework.DriverCore;
+using Core_Framework.Reporter;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+using OpenQA.Selenium;
+
+namespace Core_Framework.NUnitTestSetup;
+
+[TestFixture]
+public class NUnit_Test_Setup
+{
+    protected IWebDriver Driver;
+    protected WebDriverBase DriverBaseAction;
+
+    protected ExtentReports? _extentReport;
+    protected ExtentTest? _extentSuite;
+    protected ExtentTest? _extentTestCase;
+
+    [OneTimeSetUp]
+    public void OneTimeSetUp()
+    {
+        HtmlReport.CreateReport();
+        HtmlReport.CreateTest(TestContext.CurrentContext.Test.ClassName)
+            .AssignAuthor("Tran Nguyen Minh")
+            .AssignDevice("PC")
+            .AssignCategory("Selenium Project");
```

Comment:
Don't be hardcoded. Please use the appsettings file or create a class to store these instances